.github/workflows/test.yml

```diff
@@ -95,3 +95,26 @@ jobs:
           path: ./build/coverage.info
           branch-min-coverage: 100
           line-min-coverage: 99
+  test-memory-statistics:
+    runs-on: ubuntu-latest
```

Comment:
Can you either print or upload the output file of this action as an artifact for verifying that this action does provide a table in the output file?